scripts/runTestsOnTravis.sh

```diff
@@ -51,6 +51,13 @@ elif [ "$1" = "js_super_cluster_tests" ]; then
 
     go test -race -v -run=TestJetStreamSuperCluster ./server -count=1 -vet=off -timeout=30m -failfast
 
+elif [ "$1" = "js_cluster_chaos_tests" ]; then
+
+    # Run JetStream clustered chaos tests. By convention, all JS cluster chaos
+    # tests with `TestJetStreamClusterChaos`.
+
+    go test -race -v -run=TestJetStreamClusterChaos ./server -count=1 -vet=off -timeout=30m -failfast
```

Comment:
Either name this `TestJetStreamChaosCluster` or then you need to add `skip_js_cluster_chaos_tests` to the "js_cluster_tests" section above...